ebml_matroska.xml

```diff
@@ -1356,18 +1356,18 @@ If set to any other value, it **MUST** match the `FileUID` value of an attachmen
     <extension type="webmproject.org" webm="1"/>
     <extension type="libmatroska" cppname="TagSimple"/>
   </element>
-  <element name="TagName" path="\Segment\Tags\Tag\+SimpleTag\TagName" id="0x45A3" type="utf-8" minOccurs="1" maxOccurs="1">
+  <element name="TagName" path="\Segment\Tags\Tag\+SimpleTag\TagName" id="0x45A3" type="utf-8" length="not 0" minOccurs="1" maxOccurs="1">
     <documentation lang="en" purpose="definition">The name of the Tag that is going to be stored.</documentation>
     <extension type="webmproject.org" webm="1"/>
   </element>
-  <element name="TagLanguage" path="\Segment\Tags\Tag\+SimpleTag\TagLanguage" id="0x447A" type="string" default="und" minOccurs="1" maxOccurs="1">
+  <element name="TagLanguage" path="\Segment\Tags\Tag\+SimpleTag\TagLanguage" id="0x447A" type="string" length="not 0" default="und" minOccurs="1" maxOccurs="1">
```

Comment:
We could add a restriction on the length which would be 3 for Matroska 1-3 and `>= 3 & <=6` for Matroska 4